src/NuGet.Status/Controllers/AdminController.cs

```diff
@@ -118,9 +118,9 @@ private async Task<bool> InsertEntityToStorage(StorageService storageService, Ma
         {
             try
             {
-                var table = storageService.GetCloudTable();
-                var operation = TableOperation.Insert(entity);
-                await table.ExecuteAsync(operation);
+                TableClient tableClient = storageService.GetTableClient();
+                await tableClient.AddEntityAsync(entity);
+
                 return true;
```

Comment:
It's sort of odd to me that we assume success if the insertion doesn't error. Both ExecuteAsync (legacy) and AddEntityAsync (new sdk) have information in the response that (in theory) could indicate a non throwing failure (HttpStatusCode on legacy TableResult and IsError property on new sdk Response).

New SDK AddEntityAsync implementation as of today will throw RequestFailedException if the REST response isn't a 201 or 204 (https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/tables/Azure.Data.Tables/src/Generated/TableRestClient.cs#L1157), so the assumption works out, but still feels sort of weird.

Just an observation. Feel free to ignore this comment.